--- server/src/services/management/customer.service.js
+++ server/src/services/management/customer.service.js
@@ -42,6 +42,15 @@
   const getCustomerOrThrow = async (id) => {
     const customer = await customerDelegate.findUnique({
       where: { MaKH: Number(id) },
+      include: {
+        Xe: {
+          include: {
+            HieuXe: true,
+            PhieuSuaChua: true,
+            PhieuThuTien: true
+          }
+        }
+      }
     });
 
     if (!customer) {
